cmd/create/machinepool/helper.go

```diff
@@ -3,14 +3,17 @@ package machinepool
 import (
 	"os"
 
+	ver "github.com/hashicorp/go-version"
```

Comment:
NP : Can you reorgnaise the imports. We try to group the imports when possible, put them into 3 seperate groups. Standard pkgs, external pkgs, and then internal pkgs, all with a space between them

Example -> https://github.com/openshift/rosa/blob/master/cmd/create/cluster/cmd.go#L20-L48

Reply:
fixed